packages/maskbook/src/extension/background-script/SteganographyService/api.ts

```diff
@@ -0,0 +1,83 @@
+import { DecodeOptions, EncodeOptions, getImageType } from '@dimensiondev/stego-js'
+import { preprocessImage } from '@dimensiondev/stego-js/esm/utils/image'
+import type { AsyncCallOptions } from 'async-call-rpc'
+import { AsyncCall } from 'async-call-rpc/full'
+import { WorkerChannel } from 'async-call-rpc/utils/web/worker'
+import { serialization } from '../../../utils/type-transform/Serialization'
+import { OnDemandWorker } from '../../../web-workers/OnDemandWorker'
+
+export let StegoWorker: OnDemandWorker | undefined
+
+if (process.env.architecture) {
+    __webpack_public_path__ = browser.runtime.getURL('/')
```

Comment:
Now this line should be removed per #2179 changes

Reply:
besides this one, everything else LGTM you can remove this line and merge it if all functions work well cc @septs